dwave/system/samplers/leap_hybrid_sampler.py

```diff
@@ -132,25 +135,26 @@ def sample(self, bqm, time_limit=None, **kwargs):
 
         Args:
             bqm (:obj:`dimod.BinaryQuadraticModel`):
-                The binary quadratic model.
+                Binary quadratic model.
 
             time_limit (int):
-                Maximum run time, in seconds, to allow the solver to work on the problem.
-                Must be at least the minimum required for the number of problem variables,
-                which is calculated and set by default.
-                The minimum time for a hybrid solver is specified as a piecewise-linear
-                curve defined by a set of floating-point pairs, the `minimum_time_limit`
-                field under
+                Maximum run time, in seconds, to allow the solver to work on the
+                problem. Must be at least the minimum required for the number of
+                problem variables, which is calculated and set by default.
+                The minimum time for a hybrid BQM solver is specified as a
+                piecewise-linear curve defined by a set of floating-point pairs,
+                the `minimum_time_limit` field under
                 :attr:`~dwave.system.samplers.LeapHybridSampler.properties`.
-                The first element in each
-                pair is the number of problem variables; the second is the minimum
-                required time. The minimum time for any particular number of variables
-                is a linear interpolation calculated on two pairs that represent the
-                relevant range for the given number of variables.
-                For example, if `LeapHybridSampler().properties["minimum_time_limit"]`
-                returns `[[1, 0.1], [100, 10.0], [1000, 20.0]]`, then the minimum time
-                for a 50-variable problem is 5 seconds, the linear interpolation of the
-                first two pairs that represent problems with between 1 to 100 variables.
+                The first element in each pair is the number of problem variables;
```

Comment:
this is way too much detail. Can we just point at the system docs for this?

Reply:
and/or put the logic in a function, like in the DQM sampler?

Reply:
I've gone for your second proposal and added a `min_time_limit` function to `LeapHybridBQMSolver`. Better to keep this description closer to the source and easier for users to understand.